tests/nvmeof/test_ceph_nvmeof_neg_tests.py

```diff
@@ -1215,9 +1461,11 @@ def run(ceph_cluster: Ceph, **kwargs) -> int:
             "CEPH-83581753": test_ceph_83581753,
             "CEPH-83581945": test_ceph_83581945,
             "CEPH-83581755": test_ceph_83581755,
+            # "CEPH-83608266": test_ceph_83608266,
         }
         operation = config["operation"]
         if operation in operation_mapping:
+            rbd_obj = initial_rbd_config(**kwargs)["rbd_reppool"]
```

Comment:
make `rbd_obj` global

Reply:
I didn’t make it global because, in my test case, `initial_rbd_config()` needs to run separately for each type of `placement_cfg`. Making it global would cause it to execute twice during the first iteration of `placement_cfg`, so I changed its visibility to local instead.